hardhat.config.js

```diff
@@ -12,6 +12,10 @@ module.exports = {
         },
     },
 
+    ignition: {
+        requiredConfirmations: 1,
+    },
```

Comment:
This is how Camino Network works. Transactions are finalized immediately (as soon as they are included in a blockc). That's why we need this, otherwise _ignition_ just waits there if there are no other transactions to create blocks. (Blocks are only created if there are new transactions)